src/client/datascience/editor-integration/cellhashprovider.ts

```diff
@@ -101,6 +81,12 @@ export class CellHashProvider implements ICellHashProvider, IInteractiveWindowLi
             .filter((e) => e.hashes.length > 0);
     }
 
+    public restartedKernel() {
```

Comment:
```suggestion
    public onKernelRestarted() {
```

As we're reacting to an event that took place (i.e. this is meant to be an event handler), I think `onKernelRestarted` is better

Reply:
Sure.

---
In reply to: [407773224](https://github.com/microsoft/vscode-python/pull/11142#discussion_r407773224) [](ancestors = 407773224)